src/screens/OtherProfile/index.js

```diff
@@ -34,30 +33,22 @@ import SpecificIssue from '../../components/Blocking/SpecificIssue';
 import StorageUtils from '../../utils/storage';
 import dimen from '../../utils/dimen';
 import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
-import useCreateChat from '../../hooks/screen/useCreateChat';
 import useFeedPreloadHook from '../FeedScreen/hooks/useFeedPreloadHook';
+import useOtherProfileScreenHooks from './hooks/useOtherProfileScreenHooks';
 import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
+import {COLORS} from '../../utils/theme';
 import {Context} from '../../context';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 import {blockUser, unblockUserApi} from '../../service/blocking';
-import {
-  checkUserBlock,
-  getOtherFeedsInProfile,
-  getOtherProfile,
-  setFollow,
-  setUnFollow
-} from '../../service/profile';
 import {downVote, upVote} from '../../service/vote';
 import {fonts, normalize} from '../../utils/fonts';
 import {generateAnonProfileOtherProfile} from '../../service/anonymousProfile';
 import {getFeedDetail} from '../../service/post';
+import {getOtherFeedsInProfile, setFollow, setUnFollow} from '../../service/profile';
 import {getSingularOrPluralText} from '../../utils/string/StringUtils';
 import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
 import {setFeedByIndex, setOtherProfileFeed} from '../../context/actions/otherProfileFeed';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useOtherProfileScreenHooks` is a good addition as it encapsulates the logic for fetching and managing other user profiles. However, I noticed that the import of `useCreateChat` has been removed. If this hook was providing necessary functionality, its removal could cause issues. Please ensure that all necessary functionality is still covered.



<!-- This is an auto-generated comment by OSS CodeRabbit -->